src/components/interactive/StackedButton.jsx

```diff
@@ -18,14 +27,15 @@ const StackedButton = ({ Icon, Link, label, disabled, isActive, hasCaret, ...res
         'transition--default',
         {
           'fdsStackedButton--disabled': disabled,
-          'fdsStackedButton--active': isActive,
+          'fdsStackedButton--isActive': isActive,
+          'fdsStackedButton--isToggled': isToggled,
         }
       )}
       disabled={disabled && Element === 'button'}
     >
       {Icon && (
         <div className="fdsStackedButton-iconWrapper">
-          <Icon size="xs" />
+          <Icon size="s" />
```

Comment:
intentional change?

Reply:
yup! Asaf wanted to increase the size of these here.